webapp/app/views.py

```diff
@@ -31,15 +31,10 @@
 
 @app.route('/', methods=['GET'])
 @app.route('/index', methods=['GET'])
+@app.route('/cluster', methods=['GET'])
```

Comment:
why this?

Reply:
because i deleted the custom cluster.html. now all pages will go through the shell html file "index.html" - the actual routing happens in index.js

Reply:
nice thx for explaining